hdf5-vol-daos.spec

```diff
@@ -197,8 +197,11 @@ done
 %endif
 
 %changelog
+* Thu Nov  5 2020 Maureen Jean <maureen.jean@intel.com> - 0.1-3.g9afaf5907
+- Update hdf5-vol and vol-test to g1d2a3acb4
```

Comment:
This doesn't seem to reflect the actual git hashes used and is only referring to a single hash when it's two different repos.